requirements.txt

```diff
@@ -1,5 +1,3 @@
-beautifulsoup4==4.9.1
-cachetools==4.2.2
-Deprecated==1.2.12
-jsonschema==3.2.0
-
+cachetools==5.5.0
```

Comment:
Why do I only see removals and no binaries added even though the versions changed?

Reply:
better question - why did all tests pass without wheels